applications/polkadot-runtime-releaser.md

```diff
@@ -0,0 +1,144 @@
+- A tool to build and manage the override WASMs.
+
+Each component will have a very detailed guide to help the chain team to set up.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Xavier Lau
+
+### Contact
+
+- **Contact Name:** Xavier Lau
+- **Contact Email:** x@acg.box
+- **Website:** https://linktr.ee/aurevoirxavier
+
+### Legal Structure
+
+Personal. (no legal structure entry)
+
+### Team's experience
+
+[Xavier Lau](https://github.com/AurevoirXavier)
+
+- Seven years of experience in Rust.
+- Four years of experience in Substrate.
+- Darwinia Network core developer, since 2019.
+- Actively at [Substrate StackExchange](https://substrate.stackexchange.com/users/251/aurevoirxavier).
+- Grants contributor, [Subalfred](https://github.com/hack-ink/subalfred), [Slothunter](https://github.com/hack-ink/slothunter).
+
+### Team Code Repos
+
+- https://github.com/hack-ink
+- https://github.com/hack-ink/AiR
+- https://github.com/hack-ink/array-bytes
+- https://github.com/hack-ink/atomicalsir
+- https://github.com/hack-ink/subalfred
+
+Please also provide the GitHub accounts of all team members.
+If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/AurevoirXavier
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/xavier-lau-b0655310a
+
+### Overview
+
+- **Total Estimated Duration:** 8 weeks
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 22,500 USD
+
+### Milestone 1 Slothunter CLI tool
+
+- **Estimated duration:** 8 weeks
+- **FTE:**  1
+- **Costs:** 22,500 USD
+
+| Number | Deliverable                   | Specification                                                                                                         |
+| -----: | ----------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+|    0a. | License                       | GPLv3                                                                                                                 |
+|    0b. | Documentation                 | There will be a guide to tell people how to use this.                                                                 |
+|    0c. | Testing guide                 | There will be a guide and a demo repository to tell the auditor how to run the tests. All components will be covered. |
+|     1. | Try-runtime CI                | Comment on a release PR and see the try-runtime result.                                                               |
+|     2. | Release CI                    | Build and release the deterministic runtimes and the node files.                                                      |
+|     3. | WASM override tool            | It can build the specify feature WASM in one step and manage them well in the corresponding folder.                   |
```

Comment:
> It can build the specify feature WASM

Could you ellaborate on that?

> manage them well in the corresponding folder.

Does that mean that the build will be stored on the github repo?

Reply:
> Does that mean that the build will be stored on the github repo?

Yes, that's pretty common. For example, https://github.com/moonbeam-foundation/moonbeam-runtime-overrides/tree/master/wasm.

Reply:
> Could you ellaborate on that?

Just as described, this is a versatile tool designed for compiling runtime with particular features. 

And I plan to utilize it for compiling the debug WASM and EVM tracing WASM within the GitHub Actions I previously referenced.